psi4/src/psi4/libfunctional/superfunctional.cc

```diff
@@ -99,6 +99,11 @@ std::shared_ptr<SuperFunctional> SuperFunctional::XC_build(std::string name, boo
     sup->set_x_alpha(xc_func->global_exchange());
     sup->set_x_beta(xc_func->lr_exchange());
 
+    // User tweakers
+    if (tweakers_->size()) {
```

Comment:
Should there be a check here for the case that `tweakers_` is uninitialized, i.e., `std::nullopt`?

Reply:
The actual use of the tweakers for this PR is all removed by now by requiring libxc v6, but it's good to future proof. What do you suggest?

Reply:
Ah, got it! I suppose that's a new feature of the code I missed. But regarding suggestions, something like:

`if (tweakers_.value().size()) {`

Will cause the code to throw an exception if `tweakers_` is uninitialized, acting as the check. If you want to avoid throwing entirely, you could instead enclose this `if` statement in another `if`-block:

`if (tweakers_.has_value()) {
...
}`

to ensure that `tweakers_` must be initialized for the `set_tweak` call to proceed.

Reply:
Ok, fixed. One doesn't want the has_value version b/c that always tries to set tweak and some/most? libxc fctls aren't tweakable.